### DISCUSSION TITLE
The End doesnt always load when switching dimensions

### DISCUSSION THREAD

**Discussion by MRBBATES1** [Mojang/minecraft-editor#28] [Bugs]
**Observed result vs. what you actually expected to happen**
It seems sometimes when I click on the End to go to the End Dimension it doesnt do anything, only Overworld and Nether seems to work.
I have to close it down and re-open for it to work. seems intermittent. 

**Preview build version:**  1.19.80.20
**Frequency (eg. Always, Sometimes, Rarely):**  Sometimes
**Local single-player, peer-to-peer, dedicated server (and how many people):** Single Player
**Graphics card:** DX12 - AMD Radeon Pro WX4100
**Device info:** 
HP Z230 SFF Workstation
Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz (8 CPUs), ~3.4GHz
Windows 10 Pro 64-bit (10.0, Build 19045) (19041.vb_release.191206-1406)
16384MB RAM

**Comment by minecraftchloe** [comment]
I think I've run into this a couple of times, too! While it's hard for me to nail down the actions I do before it happens, I _think_ it seems to happen if I've moved around 10k or so blocks in any direction in the Overworld first, and then go to the End. Any chance you remember what you were doing before you swapped dimensions that could help us narrow down more repro steps?
